indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/pki/AlternativeNames.kt

```diff
@@ -132,13 +131,13 @@ private constructor(private val extensions: List<Asn1Element>) {
  * Enumeration of implicit tags used to indicate different `SubjectAltName`s
  */
 object SubjectAltNameImplicitTags {
-    val otherName = 0uL.toImplicitTag()
-    val rfc822Name = 1uL.toImplicitTag()
-    val dNSName = 2uL.toImplicitTag()
-    val x400Address = 3uL.toImplicitTag()
-    val directoryName = 4uL.toImplicitTag()
-    val ediPartyName = 5uL.toImplicitTag()
-    val uniformResourceIdentifier = 6uL.toImplicitTag()
-    val iPAddress = 7uL.toImplicitTag()
-    val registeredID = 8uL.toImplicitTag()
+    val otherName = Asn1.ImplicitTag(0uL)
+    val rfc822Name = Asn1.ImplicitTag(1uL)
+    val dNSName = Asn1.ImplicitTag(2uL)
```

Comment:
spelling: `dnsName`

Reply:
this is intentional: it uses the same off-casing as the RFC that specifies it